### PR TITLE
Function to analyze lesions #1351

### PR DESCRIPTION
**Pull request by charleygros**
### Requirements

* Does the pull request need an update of the documentation ?
TODO in SourceForge?

### Applicable Issues

Implements #1351

### CODE REVIEW

File: scripts/sct_analyze_lesion.py [hunk 1]
```diff
@@ -232,6 +232,8 @@ def _measure_volume(self, im_data, p_lst, idx):
         printv('  Volume : ' + str(round(vol_tot_cur, 2)) + ' mm^3', self.verbose, type='info')
 
     def _measure_length(self, im_data, p_lst, idx):
+        print len(self.angles)
```
Comment (peristeri) [inline scripts/sct_analyze_lesion.py]: Are these print statements required? If so, you can use sct_utils.printv instead.

Comment (charleygros) [reply scripts/sct_analyze_lesion.py]: Removed! Is it now correct?

**Comment by peristeri** [comment]
@charleygros was there a test file part of this pull request?

**Comment by peristeri** [comment]
@benjamindeleener @jcohenadad @charleygros is the design discussion on analysing lesions completed?

**Pull request merged**